.gitlab-ci.yml

```diff
@@ -127,7 +127,6 @@ build/cuda101/openmpi/gcc/all/debug/shared:
     BUILD_HIP: "ON"
     BUILD_TYPE: "Debug"
     BUILD_PAPI_SDE: "ON"
-    RUN_EXAMPLES: "ON"
```

Comment:
Just want to note that there also needs `GINKGO_BUILD_EXAMPLES` to be set. I think currently this is done a bit implicitly, but hard-coding it to `ON` in the build scripts. But I guess it could be handled more explicitly with presets.